nifty-core/src/main/java/com/facebook/nifty/core/NettyServerTransport.java

```diff
@@ -74,6 +75,8 @@
     private final NettyServerConfig nettyServerConfig;
     private final ChannelStatistics channelStatistics;
 
+    private AtomicReference<SSLServerConfiguration> sslConfiguration;
```

Comment:
Why atomic? Since we're only using .set and .get, we can use a regular reference, right?